joint_limits_interface/include/joint_limits_interface/joint_limits_interface.h

```diff
@@ -433,22 +433,25 @@ class VelocityJointSaturationHandle
     using internal::saturate;
 
     // Velocity bounds
-    double vel_low;
-    double vel_high;
+    double vel_low = -limits_.max_velocity;
+    double vel_high = limits_.max_velocity;
+
+    if (limits_.has_position_limits)
+    {
+      // Velocity bounds depend on the velocity limit and the proximity to the position limit.
+      const double pos = jh_.getPosition();
+      vel_low = saturate(limits_.min_position - pos, -limits_.max_velocity, limits_.max_velocity);
+      vel_high = saturate(limits_.max_position - pos, -limits_.max_velocity, limits_.max_velocity);
```

Comment:
these two lines seem like they have incorrect behavior... I think you copied these from the [PositionJointSoftLimitsHandle](https://github.com/airballking/ros_control/blob/23e45dd558d24c40c6646100691b33182bf6163c/joint_limits_interface/include/joint_limits_interface/joint_limits_interface.h#L210) but flipped the subtraction of the position and position limit, resulting in incorrect units being used / arbitrary values. 

I would assume you want a hard-stop position limit like how the effort is implemented, [here](https://github.com/airballking/ros_control/blob/23e45dd558d24c40c6646100691b33182bf6163c/joint_limits_interface/include/joint_limits_interface/joint_limits_interface.h#L293)